### PR TITLE
pass modalities and tasks lists from dataset when creating the prior …

### PR DESCRIPTION
**Pull request by SergheiMihailov**
…pool

### CODE REVIEW

File: adapt_med_seg/models/lightning_model.py [hunk 1]
```diff
@@ -18,7 +23,7 @@ def __init__(
         self.modalities = modalities
 
         config = SegVolConfig(test_mode=test_mode)
-        self._model = get_model(model_name, config, **kwargs)
+        self._model = get_model(model_name, config, modalities, tasks**kwargs)
```
Comment (MJHamar) [inline adapt_med_seg/models/lightning_model.py:26]: missing put a `,` between tasks and kwargs

File: adapt_med_seg/pipelines/evaluate.py [hunk 2]
```diff
@@ -60,7 +55,16 @@ def __init__(self, **kwargs) -> None:
...
             max_val_samples=None, # never validate in eval loop
             max_test_samples=self._max_test_samples,
+
+
+        self._model = get_model(
+            model_name=self.model_name,
+            config=SegVolConfig(test_mode=True),
+            modalities=self._dataset.modalities,
+            tasks=self._dataset.labels.values(),
+            kwargs=kwargs,
```
Comment (MJHamar) [inline adapt_med_seg/pipelines/evaluate.py:65]: pretty sure this need to be **kwargs

**Review by MJHamar — COMMENTED** [review]
Can we also remove the dynamic prior initialization? We want to know if an unexpected task is encountered.

**Comment by fzsomb** [comment]
I like pool

**Pull request merged**